app/components/ui/Cards/OppEventCard.tsx

```diff
@@ -1,35 +1,41 @@
 import React from "react";
 import { Button } from "components/ui/inline/Button/Button";
+import { StrapiModel } from "models/Strapi";
 import styles from "./OppEventCard.module.scss";
-import { OppEventDate } from "./OppEventDate";
+import { FormattedDate } from "./FormattedDate";
 
 interface OppEventCardProps {
-  slug: string;
-  image: string;
-  title?: string;
-  startDate?: Date;
-  endDate?: Date;
+  details: {
+    id: string;
+    imageUrl: string | null;
+    title: string;
+    calendarEvent: StrapiModel.CalendarEvent;
+  };
   sectionType: "event" | "opportunity";
 }
 
 export const OppEventCard = (props: OppEventCardProps) => {
-  const { title, slug, startDate, endDate, image, sectionType } = props;
+  const {
+    details: { title, id, calendarEvent, imageUrl },
+    sectionType,
+  } = props;
+  // TODO: waiting on design
+  const url = imageUrl || "/placeholder.img";
```

Comment:
Is this going to break the page if there's no image? If so, can we wait to merge this until we have a placeholder image to put in?

Reply:
Ha oops, I did something like this but forgot to update this line. 

![image](https://github.com/user-attachments/assets/d5e1f26c-9870-40dc-a516-7ba8f1544aa3)

Reply:
@adriencyberspace here's the commit I forgot to push that adds a placeholder: https://github.com/Exygy/askdarcel-web/pull/93/commits/47ca97afc6f54ee7080c8fc58463cbf30ee5bf5b